2019-08-17 14:32:22,108 ERROR arian : no such table: gallery 
[in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:37 in gallery]
2019-08-25 13:30:14,893 ERROR arian : Could not build url for endpoint 'auth.logout'. Did you mean 'auth.logout_page' instead? [in /Users/elinashapiro/Documents/FLAG/flag/app.py:59 in home]
2019-08-25 13:34:26,427 ERROR arian : unexpected '}', expected ')' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:37 in gallery]
2019-08-26 22:15:17,649 ERROR dbajollari1@gmail.com : Unauthorized!!!! [in /Users/elinashapiro/Documents/FLAG/flag/events/routes.py:19 in event]
2019-08-26 22:15:18,160 ERROR dbajollari1@gmail.com : Unauthorized!!!! [in /Users/elinashapiro/Documents/FLAG/flag/events/routes.py:19 in event]
2019-08-26 22:15:18,542 ERROR dbajollari1@gmail.com : Unauthorized!!!! [in /Users/elinashapiro/Documents/FLAG/flag/events/routes.py:19 in event]
2019-08-26 22:15:19,040 ERROR dbajollari1@gmail.com : Unauthorized!!!! [in /Users/elinashapiro/Documents/FLAG/flag/events/routes.py:19 in event]
2019-08-27 20:30:07,655 ERROR arian : 'User' object has no attribute 'userEmail' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2019-08-27 20:30:11,767 ERROR arian : 'User' object has no attribute 'userEmail' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2019-08-27 20:30:49,493 ERROR arian : 'User' object has no attribute 'em' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2019-08-27 20:30:55,204 ERROR arian : 'User' object has no attribute 'em' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2019-08-27 20:31:04,565 ERROR arian : 'User' object has no attribute 'em' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2019-09-07 18:29:25,253 ERROR  : (sqlite3.OperationalError) no such column: flasklogin-users.website
[SQL: SELECT "flasklogin-users".id AS "flasklogin-users_id", "flasklogin-users"."firstName" AS "flasklogin-users_firstName", "flasklogin-users"."lastName" AS "flasklogin-users_lastName", "flasklogin-users".email AS "flasklogin-users_email", "flasklogin-users".password AS "flasklogin-users_password", "flasklogin-users".phone AS "flasklogin-users_phone", "flasklogin-users".website AS "flasklogin-users_website", "flasklogin-users"."userRole" AS "flasklogin-users_userRole", "flasklogin-users"."created_On" AS "flasklogin-users_created_On", "flasklogin-users".last_login AS "flasklogin-users_last_login", "flasklogin-users"."memberStartDate" AS "flasklogin-users_memberStartDate", "flasklogin-users"."memberExpireDate" AS "flasklogin-users_memberExpireDate" 
FROM "flasklogin-users" 
WHERE "flasklogin-users".id = ?]
[parameters: ('1',)]
(Background on this error at: http://sqlalche.me/e/e3q8) [in /Users/elinashapiro/Documents/FLAG/flag/app.py:15 in home]
2019-09-07 18:34:47,024 ERROR  : (sqlite3.OperationalError) no such column: flasklogin-users.website
[SQL: SELECT "flasklogin-users".id AS "flasklogin-users_id", "flasklogin-users"."firstName" AS "flasklogin-users_firstName", "flasklogin-users"."lastName" AS "flasklogin-users_lastName", "flasklogin-users".email AS "flasklogin-users_email", "flasklogin-users".password AS "flasklogin-users_password", "flasklogin-users".phone AS "flasklogin-users_phone", "flasklogin-users".website AS "flasklogin-users_website", "flasklogin-users"."userRole" AS "flasklogin-users_userRole", "flasklogin-users"."created_On" AS "flasklogin-users_created_On", "flasklogin-users".last_login AS "flasklogin-users_last_login", "flasklogin-users"."memberStartDate" AS "flasklogin-users_memberStartDate", "flasklogin-users"."memberExpireDate" AS "flasklogin-users_memberExpireDate" 
FROM "flasklogin-users" 
WHERE "flasklogin-users".id = ?]
[parameters: ('1',)]
(Background on this error at: http://sqlalche.me/e/e3q8) [in /Users/elinashapiro/Documents/FLAG/flag/app.py:15 in home]
2019-09-07 18:38:15,796 ERROR  : (sqlite3.OperationalError) no such column: flasklogin-users.website
[SQL: SELECT "flasklogin-users".id AS "flasklogin-users_id", "flasklogin-users"."firstName" AS "flasklogin-users_firstName", "flasklogin-users"."lastName" AS "flasklogin-users_lastName", "flasklogin-users".email AS "flasklogin-users_email", "flasklogin-users".password AS "flasklogin-users_password", "flasklogin-users".phone AS "flasklogin-users_phone", "flasklogin-users".website AS "flasklogin-users_website", "flasklogin-users"."userRole" AS "flasklogin-users_userRole", "flasklogin-users"."created_On" AS "flasklogin-users_created_On", "flasklogin-users".last_login AS "flasklogin-users_last_login", "flasklogin-users"."memberStartDate" AS "flasklogin-users_memberStartDate", "flasklogin-users"."memberExpireDate" AS "flasklogin-users_memberExpireDate" 
FROM "flasklogin-users" 
WHERE "flasklogin-users".id = ?]
[parameters: ('1',)]
(Background on this error at: http://sqlalche.me/e/e3q8) [in /Users/elinashapiro/Documents/FLAG/flag/app.py:15 in home]
2019-10-13 14:19:04,369 ERROR  : Could not build url for endpoint 'auth.profile'. Did you mean 'auth.user' instead? [in /Users/elinashapiro/Documents/FLAG/flag/app.py:15 in home]
2019-10-14 19:18:03,722 ERROR  : Encountered unknown tag 'endfor'. You probably made a nesting mistake. Jinja is expecting this tag, but currently looking for 'elif' or 'else' or 'endif'. The innermost block that needs to be closed is 'if'. [in /Users/elinashapiro/Documents/FLAG/flag/app.py:21 in home]
2019-10-14 20:59:52,447 ERROR  : no such table: siteamiccontent [in /Users/elinashapiro/Documents/FLAG/flag/app.py:21 in home]
2019-10-19 12:37:30,029 ERROR  : expected token 'end of print statement', got '=' [in /Users/elinashapiro/Documents/FLAG/flag/app.py:22 in home]
2020-01-04 20:32:56,696 ERROR  : (sqlite3.OperationalError) no such column: flagusers.membershipStatus
[SQL: SELECT flagusers.id AS flagusers_id, flagusers."firstName" AS "flagusers_firstName", flagusers."lastName" AS "flagusers_lastName", flagusers.email AS flagusers_email, flagusers.password AS flagusers_password, flagusers.phone AS flagusers_phone, flagusers.website AS flagusers_website, flagusers."userRole" AS "flagusers_userRole", flagusers."created_On" AS "flagusers_created_On", flagusers.last_login AS flagusers_last_login, flagusers."memberStartDate" AS "flagusers_memberStartDate", flagusers."memberExpireDate" AS "flagusers_memberExpireDate", flagusers."membershipStatus" AS "flagusers_membershipStatus" 
FROM flagusers 
WHERE flagusers.id = ?]
[parameters: ('1',)]
(Background on this error at: http://sqlalche.me/e/e3q8) [in /Users/elinashapiro/Documents/FLAG/flag/app.py:22 in home]
2020-01-12 21:27:22,975 ERROR arian : no such table: userflag [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2020-01-12 21:43:06,378 ERROR arian : 'photo' is undefined [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2020-01-12 21:43:06,379 ERROR arian : 'photo' is undefined [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2020-02-09 21:32:23,812 ERROR arian : expected token 'end of statement block', got '=' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2020-02-09 21:32:23,813 ERROR arian : expected token 'end of statement block', got '=' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2020-02-09 21:32:26,338 ERROR arian : expected token 'end of statement block', got '=' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2020-02-09 21:33:18,082 ERROR arian : expected token 'end of statement block', got '=' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2020-02-09 21:49:06,680 ERROR arian : expected token 'name', got '|' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2020-02-09 21:49:06,680 ERROR arian : expected token 'name', got '|' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
2020-02-09 21:55:27,162 ERROR arian : expected token 'name', got '|' [in /Users/elinashapiro/Documents/FLAG/flag/gallery/routes.py:44 in gallery]
